tests/serialization/test_file_link.py

```diff
@@ -0,0 +1,23 @@
+"""Tests of FileLink serialization and deserialization."""
+import pytest
+
+from citrine.resources.file_link import FileLink
+from tests.utils.factories import FileLinkDataFactory
+
+
+@pytest.fixture
+def valid_data() -> dict:
+    return FileLinkDataFactory(url='www.citrine.io', filename='materials.txt')
```

Comment:
Since this is a one-liner, I would recommend just ditching the fixture altogether and having each test call the factory.  This helps the test case tell the story better.

Reply:
Done